tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,82 @@
+import { BooleanLike } from 'common/react';
+import { useBackend } from '../backend';
+import { Icon, Section, Stack } from '../components';
+import { Window } from '../layouts';
+
+interface CrewCreditsData {
+  groups: GroupBlockProps[];
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600}>
+      <Window.Content scrollable>
+        {data.groups?.map(
+          (group, index) =>
+            data.groups[index].crew.length > 0 && <GroupBlock key={index} group={group.group} crew={group.crew} />
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+interface GroupBlockProps {
+  group: string;
+  crew: CrewMemberProps[];
+}
+
+const GroupBlock = (props: GroupBlockProps) => {
+  const { group: group_title, crew } = props;
+
+  const heads = crew?.filter((member) => member.head);
+  const non_heads = crew?.filter((member) => !member.head);
+
+  return (
+    <Section title={group_title}>
+      <Stack fill vertical>
+        {heads?.map((member, index) => (
+          <CrewMember
+            key={'head' + index}
+            real_name={member.real_name}
+            dead={member.dead}
+            player={member.player}
+            role={member.role}
+            head
+          />
+        ))}
+        {non_heads?.map((member, index) => (
+          <CrewMember
+            key={index}
+            real_name={member.real_name}
+            dead={member.dead}
+            player={member.player}
+            role={member.role}
+          />
+        ))}
```

Comment:
> Prefer `filter` or `find` vs. two `map`s.

i'll be honest with you, we kinda need to have 2 maps (as far as i know) in case some department inexplicably has multiple heads